Configuration/TCA/Overrides/tt_content.php

```diff
@@ -59,7 +59,7 @@
 ];
 
 // use the same configuration for visible backend fields as "Textmedia"
-if (!$GLOBALS['TCA']['tt_content']['defaultTypeConfiguration']) {
+if (!isset($GLOBALS['TCA']['tt_content']['defaultTypeConfiguration'])) {
     $showItemParts = explode('--div--;LLL:EXT:core/Resources/Private/Language/Form/locallang_tabs.xlf:language,', $GLOBALS['TCA']['tt_content']['types']['textmedia']['showitem']);
     $GLOBALS['TCA']['tt_content']['defaultTypeConfiguration']['begin'] = '--palette--;LLL:EXT:frontend/Resources/Private/Language/locallang_ttc.xlf:palette.general;general,';
```

Comment:
maybe we should never write a GLOBAL TCA Configuration in this case, but only configure the showitem of the cta CE? (but this will be a breaking change in some case)

Reply:
Yes, makes sense to just write a single `showitem` configuration for the cta element.